### PR TITLE
Paradise's soul removal part one of many: Captain's drip and resprite

### PR DESCRIPTION
**Pull request by ElorgRHG**
## What Does This PR Do

Port of _some_ stuff from https://github.com/tgstation/tgstation/pull/64377

Resprites the entirety of captain's uniforms, jumpsuits, suits and caps. Their unique EVA suit was left untouched.

## Why It's Good For The Game

So I have decided that Paradise's sprites are showing their age... badly, and thus with all the crusading after soulful sprites in this community, its time this codebase has joined the fight, and its led by an amateur coderspriter, yep that's me, so expect the highest of quality. 

But anyways, the captain, in my opinion, should look _good_, and _flashy_, and I feel these sprites do justice to that opinion.

Now you may ask, why change these sprites, our old and soulful sprites are just fine, to which _I won't respond_. If you don't innovate and don't try new things, you'll never grow, or grow stagnant, and that's a bad bad bad thing.

## Images of changes

I will be showing just how these sprites look on humans below, Vox, drask, grey, _**kidan**_ sprites are also in this PR but for the sake of my sanity, won't be arranging the screenshots for them.

Suits:

<details>
If you leave a comment about feet I will find you, and I will kill you.

image

image

image

image

</details>

Jumpsuits:

<details>

image

image

image

image

</details>

Hats:

<details>

image

image

</details>

## Testing

Sign up as captain, don't look like I'm wearing a sprite from 2010 or earlier, wow.
Check my garment bag in my locker, they are also not sprites from 2010 or earlier, wow.

## Changelog
:cl: ElorgRHG, Christasmurf 
add: Captain's berets, blue and white variants.
tweak: Captain's suit clothes, carapace, tunic, etc. can now hold the disk, stamps, flasks, melee weapons, the lockbox, flashes, matches, lighters, cigarettes and emergency internals tanks.
imageadd: Resprited captain's uniforms, suits and hats.
/:cl:

_Beware sec mains, I'm coming for your drip next_

### CODE REVIEW

**Comment by LightFire53** [comment]
I was hoping that we would have kept the white parade tunic and cap, I'll be honest. The rest of it looks alright, though I'm by no means an authority on spriting

**Comment by ElorgRHG** [comment]
Going to be honest but hold myself back a bit, the white cap and tunic looked _bad_ and I've not seen a single person wear that combo, or one of them, consistently. The white color didn't even fit the captain's scheme at all.

I'll quote this from the TG's PR:
>"But Positiiiiive, where's the old white captain tunic?"

>Glad you asked because I nuked it to oblivion. In its stead, I've made a new jacket for captains!
>The logic behind this decision was that it looked out of place in the captain's wardrobe, so I opted for this instead, not only for consistency, but for something simple and serious, and rather stylish! (It is also a reference :) )

**Comment by Contrabang** [comment]
I personally think these sprites are way to bright. I know almost no one uses the tunic but plenty of players (including myself) still use the parade cap, jacket, (and perhaps less often the uniform). I think the HOS uniform on the original PR looks nice but I think the others don't fit very well.

**Comment by ElorgRHG** [comment]
> I personally think these sprites are way to bright.

How did you come to that conclusion? Old on the left, new on the right

<details>

image

image

</details>

Maybe I handpicked the best examples to reinforce my point more but most of these changes is changing the hue of the blue so it doesn't look like it was sitting in a shed for a couple decades, withering away. Its a more pleasant blue in my opinion.

**Comment by Contrabang** [comment]
Perhaps its just the slight saturation/hue change, but it does make them look brighter to me. I guess they are fine, but I personally prefer the old captain's cap over the new one. It felt unique compared to this one which feels pretty generic, especially when compared to the ones given to the HOS/Centcom(?) on the original PR.

**Comment by ElorgRHG** [comment]
> t felt unique compared to this one which feels pretty generic, especially when compared to the ones given to the HOS/Centcom(?) on the original PR.

Did it _really_? You can't tell me with a straight face that the old peaked cap looked better, and if you do, I won't believe you.
And also, the HoS and CC versions in the TG PR are literally the same peaked cap design but red and green respectively.
image

**Comment by Contrabang** [comment]
> Did it _really_? You can't tell me with a straight face that the old peaked cap looked better, and if you do, I won't believe you.

The old peaked cap looks better :P

**Comment by Contrabang** [comment]
Also to be clear, my issue with the new cap is the shape, not the colors. I just think the shape of the old cap had more uniqueness and definition. And 
> the HoS and CC versions in the TG PR are literally the same peaked cap design but black+red and green respectively.

is my exact problem with that part, just making it a recolor instead of a unique item.

**Comment by Bm0n** [comment]
The old peaked cap is missing its visor, you can't even call it a peaked cap and TBH looks god awful shape wise.

**Comment by ElorgRHG** [comment]
> he cutouts for the hands on the human version are messed up, compare the left side to the old one.

Fixed:tm:

**Comment by ElorgRHG** [comment]
Scroll down and you shall receive, captain beret has been added.

image

image

**Comment by matttheficus** [comment]
Captain’s Beret alone is a godly add.

**Comment by Necaladun** [comment]
'd like to leave things like the parade tunic as a 'classic' outfit. More options for drip is generally straight bussin', fr.

**Comment by ElorgRHG** [comment]
> 'd like to leave things like the parade tunic as a 'classic' outfit. More options for drip is generally straight bussin', fr.

If I'm reading this correctly and you want to campaign for the white tunic to remain, yeah no. Read my first comment on this PR, 
but also
a point of a resprite is to replace the bad looking sprites, with ones that look good (more often than not). And I'm not a supporter of hoarding sprites just because.  As such I will not be adding these sprites and keeping the old, shitty looking white tunic that I've never seen used, since the ceasation of my 6 month break from Paradise, and before it.

The white doesn't even fit the captain's color palette, it detracts from it, its wearer is made less recognisable as a captain, because the captain is associated with the colors blue and gold, not white. Would fit the CMO more if we're talking about that.

So to summarise, no, the white tunic will not remain in this PR, the resprited blue one will stand in its place. If you care enough about the right of that one person that unironically uses it to have negative drip, re-add it in a different PR, because I don't. 

If the removal of the white tunic is the thing that will close this PR, then well, RIP captain mains' drip, I'm moving on to respriting security.


Also zoomer language makes my soul shatter into pieces, it hurts.

**Comment by Necaladun** [comment]
I'm not trying to 'campaign' for anything. I'm putting it as a condition of me voting on this PR, because I want it to remain.

Thus, I'll be voting against it. Please try to be less of a dick in your responses and work in good faith with us, especially over such simple requests. We have enough people dying on stupid hills.

**Comment by Christasmurf** [comment]
Just add the new sprite as a different item and leave the white parade tunic as it's own thing or something

**Comment by ElorgRHG** [comment]
I've realised that I've not posted the old white tunic here before, so this is the sprite that some people want for it to remain
image

quite honestly, I don't see the appeal, its a crusty, old, distasteful and a sprite from a clearly different era of spriting.

But I've decided on a compromise that might satisfy the fans of the color white
image

a white _modern_ tunic, and a white beret, because fuck the CE and their white beret exclusivity.

**Comment by Necaladun** [comment]
That's a nice jacket/tunic for sure - but it's a very different piece of clothing compared to the full coat. It seems easiest to just leave the full coat in.

**Comment by LightFire53** [comment]
It's the shape of the old tunic I like. It could get a colour touch up, absolutely but the new variant proposed is a very different shape

**Comment by Necaladun** [comment]
> It's the shape of the old tunic I like. It could get a colour touch up, absolutely but the new variant proposed is a very different shape

Yeah, it could certainly be made better, but I would call it a full coat rather than a tunic. It has a very formal feel, like what you'd wear while officiating a wedding.

**Comment by ElorgRHG** [comment]
White tunic part two, electric boogaloo, courtesy of @Christasmurf and _mildly_ modified

tunic whoite


With this, beside the armor values, which I'd like for someone to look at them, this PR is feature complete I feel.

File: code/modules/clothing/head/jobs.dm [hunk 1]
```diff
@@ -17,20 +17,37 @@
...
+	icon_state = "captain_cap"
+
+/obj/item/clothing/head/caphat/beret
+	name = "captain's beret"
+	desc = "For Captains that are more inclined towards style."
+	icon_state = "captain_beret"
+	sprite_sheets = list(
+		"Vox" = 'icons/mob/clothing/species/vox/head.dmi',
+		"Drask" = 'icons/mob/clothing/species/drask/head.dmi',
+		)
```
Comment (hal9000PR) [inline code/modules/clothing/head/jobs.dm:45]: would you make a loadout option for these items?

Comment (ElorgRHG) [reply code/modules/clothing/head/jobs.dm:45]: Just berets or all of them?

Comment (ElorgRHG) [reply code/modules/clothing/head/jobs.dm:45]: Added the berets to the loadout.

Adding everything else would clutter too much I feel.

**Review by hal9000PR — APPROVED** [review]
